docs/layers.md

```diff
@@ -1148,7 +1148,7 @@ Railway `service` values are:
 
 ![image](images/mapzen-vector-tile-docs-roads-piers.png)
 
-**Piers** start showing up at zoom 13+ with `kind_detail` values of `pier`.
+**Piers** start showing up at zoom 11+ with `kind_detail` values of `pier`.
```

Comment:
This is the road layer docs, please reset.

We generally don't indicate which zooms for which landuse so I'd leave this change undocumented.